### ISSUE TITLE
Listen der Rechnungen/Belege in der SPA

### ISSUE THREAD

**Issue by d135-1r43** [hopps-app/hopps#155]
# Story

As a **Bommelwart**, I want to view invoices as a list, including their sums, and have the ability to filter and sort them efficiently, so I can easily analyze and manage the data.

# Acceptance Criteria

1. **List Display**:
   - The invoices are displayed in a table using **AG Grid Community Edition**.
   - Each row represents an individual invoice, and the columns display relevant details such as date, creditor, submitter, and amount.
2. **Filtering**:
   - Filters should be **Excel-like** with options for:
     - Time span
     - Creditor
     - Submitter
     - Amount
   - Filtering by **Bommel**:
     - A recursive filter via an external tree view of all Bommels must be available, where the Bommels can be multiselected to filter
3. **Sorting**:
   - Every column must support ascending and descending sorting.
4. **Summation**:
   - A summary row or section must display the total sum of the filtered invoices.

## Technical Notes
- The tree view for Bommel filtering can be implemented as an independent component, but must integrate seamlessly with the AG Grid filters.
- Sorting and filtering must work together to reflect the user's selected criteria without conflicts.

**Comment by d135-1r43** [comment]
Blocked by #109

**Comment by kzamurnyak** [comment]
blacked by backed

**Comment by kzamurnyak** [comment]
Hey. @d135-1r43 
i see that information what we have in our backend does not match with the model on backend:
<img width="315" alt="Image" src="https://github.com/user-attachments/assets/6b42a8bd-f66f-4343-b914-5b7c14ea7184" />

So, i'm not sure what should be displayed in this table (for example we don't have Creditor and Submitter here).

**Comment by d135-1r43** [comment]
You are right. Let's use what we have now. The problem is, that a model change will require changes at nearly every layer, so we really need to get this done right. 

@manuelhummler, what are your needs coming from mobile?
